packages/calypso-e2e/src/lib/utils/get-test-account-by-feature.ts

```diff
@@ -4,10 +4,10 @@ import type { JetpackTarget, SupportedEnvVariables } from '../../types/env-varia
 
 export type TestAccountEnvVariables = Pick<
 	SupportedEnvVariables,
-	'GUTENBERG_EDGE' | 'COBLOCKS_EDGE' | 'TEST_ON_ATOMIC' | 'JETPACK_TARGET'
+	'GUTENBERG_EDGE' | 'GUTENBERG_NIGHTLY' | 'COBLOCKS_EDGE' | 'TEST_ON_ATOMIC' | 'JETPACK_TARGET'
 >;
 
-type Env = 'edge' | 'stable';
+type Env = 'edge' | 'stable' | 'nightly';
```

Comment:
**Suggestion (non-blocking / take-it-or-leave-it 😄 ):** Out of curiosity, what do you think of collapsing these two ENV vars `GUTENBERG_EDGE` and `GUTENBERG_NIGHTLY` into just a single ENV var like `GUTENBERG_TARGET` that has these three values? 

Totally optional of course, and I realize this may be super annoying to actually implement 😆 (because you'd have to tweak the old Gutenberg TC builds), but I think it might read a bit nicer and scale a bit cleaner. We took this approach with Jetpack and it's worked well there so far! I just sometimes find combining multiple booleans into one enum-like type to sometimes be a bit harder to follow, because there's often implicit hierarchical rules involved. 😄

Reply:
@dpasque Jinx! I actually did that in this (somewhat old) PR: https://github.com/Automattic/wp-calypso/pull/62543. I will add a note to revisit it soon and extract that part into a smaller PR. I think there are other improvements there that should be revisited, but the PR is too big and complex, so I need to split it, and refresh my memory about it, as it's been a while!

Reply:
There's also this https://github.com/Automattic/wp-calypso/pull/62235, which is an experiment to add a wildcard syntax sugar, which could reduce the number of current repeat rules (for example, in this change, we needed to add expand the combination of nightlies to the coblocks rules and the siteEditor rules in addition to the general rule. In theory, wildcards would allow a single nightlies rule, and we could "ignore" the other attribute's values.

Reply:
Actually! I think https://github.com/Automattic/wp-calypso/pull/62543 implements a simpler alternative to wildcards 🤔 -- anyway, I'll try to revive it soon, giving priority to your suggestion of simplifying the way we pass the gutenberg version.